addons/website_blog/static/src/snippets/s_latest_posts/000.scss

```diff
@@ -1,4 +1,4 @@
-.s_latest_posts, .s_latest_posts_big_picture {
+.s_latest_posts:not([data-vcss]), .s_latest_posts_big_picture {
```

Comment:
Missing compatibility selector on second class

Reply:
The second class is no longer used in XML and thus there is no versioned css that can affect it. I can add it for consistency if you would like, or add a comment explaining why it isn't versioned.